packages/css/src/components/radio/radio.scss

```diff
@@ -44,111 +23,181 @@
+
+    .ams-radio__icon circle:nth-child(2) {
+      // TODO: currently disabled invalid gets the same styling as disabled. This should get its own styling.
+      fill: var(--ams-radio-icon-disabled-invalid-inner-fill);
+    }
   }
 }
 
-// HOVER STATES
+// HOVER
 
-// Invalid unchecked hover
-.ams-radio__input[aria-invalid="true"] + .ams-radio__label:hover .ams-radio__circle::after {
-  // TODO: this should be the (currently non-existent) dark red hover color
-  border-color: var(--ams-radio-circle-invalid-hover-border-color);
+// Disabled label hover
+.ams-radio__input:disabled + .ams-radio__label:hover {
+  text-decoration: none;
 }
 
-// Default checked hover
-.ams-radio__input:checked + .ams-radio__label:hover .ams-radio__circle::after {
-  background-image: var(--ams-radio-circle-checked-hover-background-image);
-}
+// Invalid hover
+.ams-radio__input[aria-invalid="true"] + .ams-radio__label:hover {
+  .ams-radio__icon circle:first-child {
+    // TODO: this should be the (currently non-existent) dark red hover color
+    stroke: var(--ams-radio-icon-invalid-hover-outer-stroke);
+  }
 
-// Invalid checked hover
-.ams-radio__input[aria-invalid="true"]:checked + .ams-radio__label:hover .ams-radio__circle::after {
-  // TODO: this should be the (currently non-existent) dark red hover color
-  background-image: var(--ams-radio-circle-invalid-checked-hover-background-image);
+  .ams-radio__icon circle:nth-child(2) {
+    // TODO: this should be the (currently non-existent) dark red hover color
+    fill: var(--ams-radio-icon-invalid-hover-inner-fill);
+  }
 }
 
-// Disabled checked hover
-.ams-radio__input:disabled:checked + .ams-radio__label:hover .ams-radio__circle::after {
-  background-image: var(--ams-radio-circle-disabled-checked-hover-background-image);
+// Disabled invalid hover
+.ams-radio__input[aria-invalid="true"]:disabled + .ams-radio__label:hover {
+  .ams-radio__icon circle:first-child {
+    // TODO: currently disabled invalid gets the same styling as disabled. This should get its own styling.
+    stroke: var(--ams-radio-icon-disabled-invalid-hover-outer-stroke);
+  }
+
+  .ams-radio__icon circle:nth-child(2) {
+    // TODO: currently disabled invalid gets the same styling as disabled. This should get its own styling.
+    fill: var(--ams-radio-icon-disabled-invalid-hover-inner-fill);
+  }
 }
 
-// Disabled invalid unchecked hover
-.ams-radio__input[aria-invalid="true"]:disabled + .ams-radio__label:hover .ams-radio__circle::after {
-  // TODO: currently disabled invalid gets the same styling as disabled. This should get its own styling.
-  border-color: var(--ams-radio-circle-disabled-border-color);
+// FORCED COLORS
+
+// Default
+@media (forced-colors: active) {
+  .ams-radio__label,
+  .ams-radio__label:hover,
+  .ams-radio__input[aria-invalid="true"] + .ams-radio__label,
+  .ams-radio__input[aria-invalid="true"] + .ams-radio__label:hover {
+    .ams-radio__icon {
+      circle:first-child {
+        stroke: FieldText;
```

Comment:
Could this be solved by `currentColor`? So you can just set the `.ams-radio__label` to `color: FieldText`? Just asking maybe you've already tried?

Reply:
You could, but the label text shouldn't really be `FieldText`, but `CanvasText`. So they're 2 different system colors, which is why `currentColor` doesn't really work here, I think.